internal/hooks/NewRound_IdleMobs.go

```diff
@@ -23,6 +24,8 @@ func IdleMobs(e events.Event) events.ListenerReturn {
 
 	// evt := e.(events.NewRound)
```

Comment:
Commented out code, should it be removed?

Reply:
Done